dashboards/numpy/numpy.json

```diff
@@ -0,0 +1,666 @@
+{
```

Comment:
Let's get the entity GUIDs and explicit times ranges removed from this dashboard!